api/base/settings/defaults.py

```diff
@@ -339,6 +340,13 @@
     'default': {
         'BACKEND': 'django.core.cache.backends.locmem.LocMemCache',
     },
```

Comment:
Do we happen to know what this cache is used for?

Reply:
Nope. But it's there so I didn't change it.

Reply:
Yeah, let's skip worrying about this one for now.